spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/onnx/OnnxSum.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.nn.onnx
+
+import scala.reflect.ClassTag
+import com.intel.analytics.bigdl.nn
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+
+
+object OnnxSum {
```

Comment:
Why do you name it as  `OnnxSum,` not `Sum` ?

Reply:
Name conflict.
There is a layer called Sum under nn, but it does not do what this Sum does.